nextcord/ext/commands/bot.py

```diff
@@ -1037,7 +1037,6 @@ async def process_commands(self, message: Message) -> None:
     async def on_message(self, message):
         await self.process_commands(message)
 
-
```

Comment:
;-; pep8?
also this looks so random

Reply:
i was messing around with stuff and committed them removed it but i guess i kept this

Reply:
yeah this wasnt...resolved?